nilearn/datasets/utils.py

```diff
@@ -14,9 +14,12 @@
 import sys
 import tarfile
 import urllib
+import socket
 import warnings
 import zipfile
 
+socket.setdefaulttimeout(8)
+
```

Comment:
Extra white space ?

Reply:
Do you think so? there should be 2 blank lines before the function that comes below, right? flake8 doesn't seem to complain. anyway, happy to change it if you prefer

Reply:
Sorry about that, you're absolutely right. Ignore this comment :smile_cat: